### PR TITLE
Update bcc.mk

### PR DESCRIPTION
**Pull request by EMG60**
Fixed to work with BCC 7.60 64 bit (CLANG)

### CODE REVIEW

**Comment by EMG60** [comment]
-

File: config/win/bcc.mk [hunk 1]
```diff
@@ -22,17 +22,19 @@ CC_OUT := -o
 
 CFLAGS += -I. -I$(HB_HOST_INC)
 
-CFLAGS += -q -tWM -CP437
```
Comment (alcz) [inline config/win/bcc.mk:25]: around here

**Comment by alcz** [comment]
Contributed commit do not have updated ChangeLog.txt so i cannot apply as is on github.

**Comment by alcz** [comment]
What about BCC 5.5? It was not deprecated in Harbour 3.2

**Comment by EMG60** [comment]
Il 25/10/2023 17:43, alcz ha scritto:

> What about BCC 5.5? It was not deprecated in Harbour 3.2

Now I'm using MSC but I used BCC since 5.5 and never needed that switches.

-- 
Enrico Maria Giordano

http://www.emagsoftware.it
http://www.emagsoftware.it/emgmusic
http://www.emagsoftware.it/spectrum
http://www.emagsoftware.it/tbosg

**Comment by tuffnatty** [comment]
> Il 25/10/2023 17:24, alcz ha scritto:
> > Contributed commit do not have updated ChangeLog.txt so i cannot apply as is on github.
> Done.

@EMG60 You've committed ChangeLog.txt and other files as separate pull requests. Please check https://github.com/harbour/core#how-to-participate , and commit the ChangeLog in the same PR with the fix being described.

**Comment by EMG60** [comment]
Il 29/10/2023 12:45, Phil Krylov ha scritto:

> @EMG60 <https://github.com/EMG60> You've committed ChangeLog.txt and 
> other files as separate pull requests. Please check 
> https://github.com/harbour/core#how-to-participate 
> and commit the ChangeLog in the same PR with the fix being described.

I delete all my previous pull requests and done a new one. Hope it is OK 
now.

-- 
Enrico Maria Giordano

http://www.emagsoftware.it
http://www.emagsoftware.it/emgmusic
http://www.emagsoftware.it/spectrum
http://www.emagsoftware.it/tbosg

**Pull request closed without merging**